client/components/site-selector/style.scss

```diff
@@ -53,16 +53,20 @@
 	}
 }
 
-
 .notouch .site-selector.is-hover-enabled .site:hover,
-.notouch .site-selector.is-hover-enabled .all-sites:hover
-.site-selector .site.is-highlighted,
+.notouch .site-selector.is-hover-enabled .all-sites:hover .site-selector .site.is-highlighted,
 .site-selector .all-sites.is-highlighted {
 	background-color: var( --color-neutral-5 );
 	cursor: pointer;
 
 	.site__badge {
 		background-color: var( --color-surface );
+
```

Comment:
This probably won't need the nesting - could be just the `site__badge-p2` selector?

Reply:
I can try that. I originally aimed for something like `.is-p2` or `.is-p2-workspace` (so instead of a full selector make it a modifier). Perhaps I'll refactor it like that.

Reply:
Oh, sorry. I was unclear. I think the naming was great – I meant that it might be unnecessary to have the css compile to  `.site__badge.site__badge-p2` so if the code in `.site__badge-p2` was move up a step in the nesting it would just be `.site__badge-p2`. I think that would be specific enough?